### PR TITLE
Feature/dact 1597 title included in name for directors service

### PR DESCRIPTION
**Pull request by cdrainCH**
JIRA: https://companieshouse.atlassian.net/jira/software/c/projects/DACT/boards/492?selectedIssue=DACT-1597

Adds title to the name for pages in the directors service.

### CODE REVIEW

**Review by mdesaikns — APPROVED** [review]

File: src/controllers/remove.director.check.answers.controller.ts [hunk 1]
```diff
@@ -23,7 +26,9 @@ export const get = async (req: Request, res: Response, next: NextFunction) => {
     const session: Session = req.session as Session;
     const companyProfile: CompanyProfile = await getCompanyProfile(companyNumber);
     const companyOfficer: CompanyOfficer = await getDirectorAndTerminationDate(session, transactionId, submissionId);
-    
+    const appointmentId = (await getOfficerFiling(session, transactionId, submissionId)).referenceAppointmentId as string
```
Comment (AndreKramer7654) [inline src/controllers/remove.director.check.answers.controller.ts:29]: should verify that appointmentId is found and not null/undefined

File: src/controllers/remove.director.check.answers.controller.ts [hunk 2]
```diff
@@ -23,7 +26,9 @@ export const get = async (req: Request, res: Response, next: NextFunction) => {
...
+    const appointment: CompanyAppointment = await getCompanyAppointmentFullRecord(session, companyNumber, appointmentId)
```
Comment (AndreKramer7654) [inline src/controllers/remove.director.check.answers.controller.ts:30]: should verify that appointment is found. I think getCompanyAppointmentFullRecord returns null on 404.

File: views/remove-director-check-answers.html [hunk 3]
```diff
@@ -40,6 +40,12 @@ <h2 class="govuk-heading-m">Company details</h2>
             }
           ]
         }) }}
+
```
Comment (AndreKramer7654) [inline views/remove-director-check-answers.html:43]: seems strange to have to add a new var name and not update directorName. Was directorName not used?

File: src/controllers/remove.director.check.answers.controller.ts (hunk 1 above)
Comment (cdrainCH) [reply src/controllers/remove.director.check.answers.controller.ts]: Fixed

File: src/controllers/remove.director.check.answers.controller.ts (hunk 2 above)
Comment (cdrainCH) [reply src/controllers/remove.director.check.answers.controller.ts:33]: Fixed

File: views/remove-director-check-answers.html (hunk 3 above)
Comment (cdrainCH) [reply views/remove-director-check-answers.html:43]: Good spot, must have been left over from a copy/paste

File: src/controllers/remove.director.check.answers.controller.ts [hunk 4]
```diff
@@ -23,7 +26,12 @@ export const get = async (req: Request, res: Response, next: NextFunction) => {
     const session: Session = req.session as Session;
     const companyProfile: CompanyProfile = await getCompanyProfile(companyNumber);
     const companyOfficer: CompanyOfficer = await getDirectorAndTerminationDate(session, transactionId, submissionId);
-    
+    const appointmentId = (await getOfficerFiling(session, transactionId, submissionId)).referenceAppointmentId as string;
```
Comment (AndreKramer7654) [inline src/controllers/remove.director.check.answers.controller.ts:29]: Remove the "as string" as not needed.

File: src/controllers/remove.director.check.answers.controller.ts (hunk 2 above)
Comment (AndreKramer7654) [reply src/controllers/remove.director.check.answers.controller.ts:33]: Turns out the getCompanyAppointmentFullRecord throws errors if not found. Which is different than the other apis so all good.

File: src/controllers/remove.director.check.answers.controller.ts (hunk 4 above)
Comment (cdrainCH) [reply src/controllers/remove.director.check.answers.controller.ts]: Removed

**Review by AndreKramer7654 — DISMISSED** [review]

**Review by mdesaikns — APPROVED** [review]

**Review by AndreKramer7654 — APPROVED** [review]

**Pull request merged**